### PR TITLE
Feat/profile management

### PR DESCRIPTION
**Pull request by RhodyArthur**

### CODE REVIEW

File: src/app/features/profile-management/components/admin-profile-info/admin-profile-info.component.ts
Comment (BlissArmah-Nwanwah) [inline src/app/features/profile-management/components/admin-profile-info/admin-profile-info.component.ts:1]: I think you unsubscribing is not working 
Kindly check how it is done in the documentation and try that

File: src/app/features/profile-management/components/change-password/change-password.component.ts
Comment (BlissArmah-Nwanwah) [inline src/app/features/profile-management/components/change-password/change-password.component.ts:1]: Handle unsubscription properly

File: src/app/features/profile-management/pages/user-profile/user-profile.component.ts
Comment (BlissArmah-Nwanwah) [inline src/app/features/profile-management/pages/user-profile/user-profile.component.ts:1]: unsubscribe from the observable

File: src/app/features/profile-management/services/image-management.service.ts [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
+import { jwtDecode } from 'jwt-decode';
+import { User } from '../models/user';
+import { decodeToken } from '../../../utils/decodeToken';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ImageManagementService {
+  // apiUrl = 'https://authservice.ahmedzubairu.xyz/api/auth/public/update-profile-picture'
+  token!: string
```
Comment (BlissArmah-Nwanwah) [inline src/app/features/profile-management/services/image-management.service.ts]: remove the comment

File: src/app/shared/components/user-profile-header/user-profile-header.component.ts
Comment (BlissArmah-Nwanwah) [inline src/app/shared/components/user-profile-header/user-profile-header.component.ts:1]: destroy the subscription here after use

**Comment by RhodyArthur** [comment]
Noted, fixing it

**Review by BlissArmah-Nwanwah — APPROVED** [review]

**Pull request merged**